src/libs/actions/App.js

```diff
@@ -341,12 +341,18 @@ function openProfile(personalDetails) {
 }
 
 function beginDeepLinkRedirect() {
+    // Match any magic link (/v/<account id>/<6 digit code>)
+    const isMagicLink = CONST.REGEX.ROUTES.VALIDATE_LOGIN.test(window.location.pathname)
```

Comment:
`window` is available in Web only and `App.js` is platform independent. Can we use `Navigation.getActiveRoute()` instead?

Reply:
That would be great!

NAB - Also, that Regex is weird. But it seems to work. Are all routes with `/v/` for validate codes?

Reply:
@s77rt `Navigation.getActiveRoute()` returns an empty string in that case. I don't think it's an issue here, as `beginDeepLinkRedirect` is only called by the website.

Reply:
@dangrous yes, magic link is the only url that has `/v` in it

Reply:
Would it be better if we move that `window` logic to `DeeplinkWrapper` then pass a param to `beginDeepLinkRedirect`/`beginDeepLinkRedirectAfterTransition` e.g:

```js
// src/libs/actions/App.js
function beginDeepLinkRedirect(shouldAuthenticateWithCurrentAccount = true) {
    ...
    if (!currentUserAccountID || !shouldAuthenticateWithCurrentAccount) {
        Browser.openRouteInDesktopApp();
        return;
    }
    ...
}
```

Reply:
I'm not sure if we need to?

Reply:
Just to keep the platform dependent code on the platform dependent file. Using `window` in a `.website.js` file looks better than using it in a regular `.js` file.

Reply:
oh i see, since it's already separated out. Yeah that makes sense!

Reply:
All right, I am on it

Reply:
Done!